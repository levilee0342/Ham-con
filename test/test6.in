-13769
-25608
-21446
1367
-13053
-5400
-22351
1626
-12290
-27605
-13940
