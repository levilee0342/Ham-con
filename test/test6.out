1626
-27605